tests/benchmark/test_osu.py

```diff
@@ -6,6 +6,57 @@
 from .. import FakeCluster
 
 
+class TestOSUOptionsAttributes(unittest.TestCase):
+    def test_categories_assignment(self):
+        benchmark = OSU()
+        # completely replace attributes
+        benchmark.attributes = dict(
+            executable='/path/to/fake',
```

Comment:
`executable` and `srun_nodes` are not relevant for the test. Can you please remove them?

Reply:
Done.